src/reducers/tests/ui.js

```diff
@@ -28,4 +28,16 @@ describe("ui reducer", () => {
     expect(updatedState.frameworkGroupingOn).toBe(value);
     expect(prefs.frameworkGroupingOn).toBe(value);
   });
+
+  it("gets the search results", () => {
+    const state = State();
+    expect(getSearchResults({ ui: state })).toBe(state.searchResults);
+  });
+
+  it("gets the symbol search results", () => {
+    const state = State();
+    expect(getSymbolSearchResults({ ui: state })).toBe(
+      state.symbolSearchResults
+    );
```

Comment:
I like treating actions as the public interface here:

here's an actions/tests/ui test
```js
  it("should toggle the visible state of project search", () => {
    const { dispatch, getState } = createStore();
    expect(getProjectSearchState(getState())).toBe(false);
    dispatch(actions.toggleProjectSearch());
    expect(getProjectSearchState(getState())).toBe(true);
  });
```